plugins/context_datalayer_reaction.inc

```diff
@@ -59,30 +59,30 @@ class context_datalayer_reaction extends context_reaction {
     }
 
     // Add a new KVP.
-    $form['data']['add'] = array(
+    $form['context_datalayer_reaction']['data']['add'] = array(
       '#type' => 'container',
     );
-    $form['data']['add']['key'] = array(
+    $form['context_datalayer_reaction']['data']['add']['key'] = array(
       '#type' => 'textfield',
-      '#title' => t('Add Key Value'),
+      '#title' => t('Add Key/ Value'),
       '#description' => t('Provides this key-value pair to the datalayer when this section is active.'),
       '#size' => 16,
       '#maxlength' => 64,
       '#value' => '',
     );
-    $form['data']['add']['value'] = array(
+    $form['context_datalayer_reaction']['data']['add']['value'] = array(
       '#type' => 'textfield',
       '#title' => t('Add Value'),
       '#title_display' => 'invisible',
       '#maxlength' => 64,
       '#value' => '',
     );
-    $form['data']['add']['button'] = array(
+    $form['context_datalayer_reaction']['data']['add']['button'] = array(
       '#type' => 'button',
-      '#value' => t('Add'),
+      '#value' => t('Add Reaction'),
```

Comment:
This isn't necessary really, I just figured that with both forms open at the same time - it hurts your eyes to figure out which is which!  This at least is a necessary step (hitting the "add" button) so it's one final simple check.